chia/wallet/trading/trade_store.py

```diff
@@ -236,6 +301,25 @@ async def get_trade_record_with_status(self, status: TradeStatus) -> List[TradeR
 
         return records
 
+    async def get_coins_of_interest_with_trade_statuses(self, trade_statuses: List[TradeStatus]) -> List[Coin]:
+        """
+        Checks DB for TradeRecord with id: id and returns it.
+        """
+        async with self.db_wrapper.reader_no_transaction() as conn:
+            cursor = await conn.execute(
+                "SELECT distinct c.coin from coin_of_interest_to_trade_record c, trade_records t "
+                "WHERE t.status in (%s) AND c.trade_id = t.trade_id" % ",".join("?" * len(trade_statuses)),
+                ([x.value for x in trade_statuses]),
```

Comment:
you don't actually have to convert the list to a tuple. the sqlite API accepts anything `Iterable`

Reply:
fixed